### ISSUE TITLE
[layer ess ]Package company-ess is unavailable. Is the package name misspelled?

### ISSUE THREAD

**Issue by BlinkD** [syl20bnr/spacemacs#1655]
spacemacs(newest develop branch) always try to fetch company-ess on startup
```
Found 1 new package(s) to install...
--> fetching new package repository indexes...
--> installing (ess):company-ess... [1/1]
Package company-ess is unavailable. Is the package name misspelled?
```
this is my package-archives , and no company-ess in package-list 
```
  (setq package-archives '(("gnu" . "http://elpa.gnu.org/packages/")
                           ("marmalade" . "https://marmalade-repo.org/packages/")
                           ("melpa" . "http://melpa.org/packages/")))
```

**Comment by TheBB** [comment]
This is fixed in develop, as far as I can tell, by #1623.

**Comment by syl20bnr** [comment]
@BlinkD Have you updated the ess package ?

**Comment by BlinkD** [comment]
It's ok now , i rm -rf .emacs.d again , and reinstall spacemacs

**Comment by syl20bnr** [comment]
Ok, clicking on [Update] button on the homepage should be correct too.

Le jeudi 21 mai 2015, 何夕 <notifications@github.com> a écrit :

> Closed #1655 <https://github.com/syl20bnr/spacemacs/issues/1655>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/issues/1655#event-311180453>.
>


-- 
-syl20bnr-
